src/Dialogs/Onboarding.cpp

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#include "Dialogs/Dialogs.h"
+#include "Dialogs/WidgetDialog.hpp"
+#include "Widget/LargeTextWidget.hpp"
+#include "Look/DialogLook.hpp"
+#include "UIGlobals.hpp"
+#include "Language/Language.hpp"
+
+#include "Screen/Layout.hpp"
```

Comment:
I think you need to add Onboarding.hpp here. That will fix the compilation failure.